packages/expo-updates/e2e/setup/project.ts

```diff
@@ -191,6 +191,13 @@ async function copyCommonFixturesToProject(
   // copy .prettierrc
   await fs.copyFile(path.resolve(repoRoot, '.prettierrc'), path.join(projectRoot, '.prettierrc'));
 
+  // Copy react-native patch
+  await fs.mkdir(path.join(projectRoot, 'patches'));
+  await fs.copyFile(
+    path.resolve(repoRoot, 'patches', 'react-native+0.75.0-rc.5.patch'),
+    path.join(projectRoot, 'patches', 'react-native+0.75.0-rc.5.patch')
+  );
```

Comment:
Necessary for e2e tests while https://github.com/facebook/react-native/pull/45560 is not merged